src/components/end-test-link/end-test-link.ts

```diff
@@ -27,6 +27,6 @@ export class EndTestLinkComponent {
 
   onTerminate = () => {
     this.terminateTestModal.dismiss();
-    this.navController.push('DebriefPage', { outcome: 'terminated' });
+    this.navController.push('DebriefPage');
```

Comment:
Remind me how the debrief knows if its a termination?

Reply:
Test report dispatches an event to set the activity code. When the debrief page loads, it retrieves this from the store via `getTestOutcome`